test-data/unit/check-newsemanal.test

```diff
@@ -2058,6 +2058,24 @@ from enum import Enum
 A = Enum('A', ['x', 'y'])
 A = Enum('A', ['z', 't'])  # E: Name 'A' already defined on line 3
 
+[case testNewAnalyzerEnumAcceptStringLiteral]
+from enum import Enum
+from typing_extensions import Literal
+
+x: Literal['ANT BEE CAT DOG'] = 'ANT BEE CAT DOG'
+Animal = Enum('Animal', x)
+
+[builtins fixtures/tuple.pyi]
+
+[case testNewAnalyzerEnumAcceptsFinalValue]
+from enum import Enum
+from typing_extensions import Final
+
+x: Final['str'] = 'ANT BEE CAT DOG'
+Animal = Enum('Animal', x)
```

Comment:
I would also check that `Animal.ANT`, `Animal.BEE`, etc are available in both test cases.